app/api/v2/handlers/operation_api.py

```diff
@@ -227,7 +227,7 @@ async def get_operation_link(self, request: web.Request):
                           }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
     @aiohttp_apispec.response_schema(LinkSchema(partial=True),
```

Comment:
Let's update this as discussed - creation of a `LinkResultSchema` in `app/api/v2/schemas` to capture the true response value of `dict(link=<Link display object>, result=<string>)`